datagateway_api/src/datagateway_api/icat/filters.py

```diff
@@ -280,6 +281,14 @@ def icat_set_limit(query, skip_number, limit_number):
         raise FilterError(e)
 
 
+class PythonICATQueryFilter(ScoringQueryFilter):
+    def __init__(self, value):
+        super().__init__(value)
+
+    def apply_filter(self, query):
+        return
+
+
```

Comment:
This class isn't needed because it features no implementation. When defining the filter in `search_api/filters.py`, you can inherit directly from `ScoringQueryFilter`.